### DISCUSSION TITLE
Is it Legal to scrape public data from TikTok? And what about it using it for commercial goals?

### DISCUSSION THREAD

**Discussion by joseluismoreira** [davidteather/TikTok-Api#495] [Q&A]
I have noticed the following part of the tiktok's ToS 

![image](https://user-images.githubusercontent.com/43185535/107154718-4e317c00-6953-11eb-895e-44f34ec73ddc.png)

Does anyone know about the legal implications? To use the data to get insights about it, but not selling directly, should be ok?

**Comment by clchera** [comment]
Hi, have you clarified the legal implications of using this? Thank you!
